ui/appui-react/src/appui-react/pickers/ViewSelector.tsx

```diff
@@ -232,9 +241,16 @@ export class ViewSelector extends React.Component<ViewSelectorProps, ViewSelecto
         else if (this.state.showUnknown)
           unknown.push(viewItem);
       });
+
+      if(this._searchInput.length > 0){
+        views3dFiltered = views3d.filter((view) => view.name!.toLowerCase().includes(this._searchInput.toLowerCase()));
+        views2dFiltered = views2d.filter((view) => view.name!.toLowerCase().includes(this._searchInput.toLowerCase()));
+        sheetsFiltered = sheets.filter((view) => view.name!.toLowerCase().includes(this._searchInput.toLowerCase()));
+        unknownFiltered = unknown.filter((view) => view.name!. toLowerCase().includes(this._searchInput.toLowerCase()));
```

Comment:
Please use `name?.toLowerCase()` instead of `!`, the ? will return undefined if the name is not defined and filter out the entry, ! will crash the application with a "unable to access `toLowerCase` of `undefined`".